pkg/kubernetes/client.js

```diff
@@ -31,10 +31,29 @@ define([
     var API_OPENSHIFT = "/oapi";
 
     var TYPE_APIS = {
-        "images" : API_OPENSHIFT,
-        "imagestreams" : API_OPENSHIFT,
+        "nodes" : {
```

Comment:
Does it need to be here? events does have a namespaced only url and is a kubernetes api,